templates/errors/submission-complete.html

```diff
@@ -0,0 +1,8 @@
+{% extends 'layouts/_base.html' %}
```

Comment:
I think the story is a bit short here on design, not you. I was imagining it to look like the other session timeout page (i.e the green box), but with different text

Reply:
I think this is to follow the other card (as it was broken out of it) which states the green panel is to be removed. https://trello.com/c/hMomUBJJ/4089-p-401-error-messages-scenario-1-and-2-m